tests.py

```diff
@@ -1,14 +1,24 @@
 from storage import Storage
 
+
 def test_add():
     pass
 
+
 def test_remove():
-    pass
+    st = Storage({'a': 1, 'b': 2})
+    key = 'b'
+    val = st.remove(key)
+    assert val == 2, "Key {} is not in the dictionary".format(key)
+    key = 'c'
```

Comment:
Are you sure you wanted to check a 'c' key, not a removed 'b' key?

Reply:
Ok